src/platform/nRF5/BLEManagerImpl.cpp

```diff
@@ -636,7 +732,10 @@ CHIP_ERROR BLEManagerImpl::EncodeAdvertisingData(ble_gap_adv_data_t & gapAdvData
 
     // Initialize the CHIP BLE Device Identification Information block that will be sent as payload
     // within the BLE service advertisement data.
-    err = ConfigurationMgr().GetBLEDeviceIdentificationInfo(deviceIdInfo);
+    // err = ConfigurationMgr().GetBLEDeviceIdentificationInfo(deviceIdInfo);
```

Comment:
Should we delete instead of commenting out? Maybe replace with TODO if this needs to be added back